src/cryptoadvance/specter/templates/includes/helpers.jinja

```diff
@@ -76,4 +76,29 @@
 
 
 
+
+
+    async function sendRequest(url, method_str, formData) {
+        if (!formData) {
+            formData = new FormData();
+        }
+        formData.append("csrf_token", "{{ csrf_token() }}")
+        d = {
+                method: method_str,
+            }
+        if (method_str == 'POST') {
+            d['body'] = formData;
+        }
+
+        const response = await fetch(url, d);
+        if(response.status != 200){
+            showError(await response.text());
+            console.log(`Error while calling ${url} with ${method_str} ${formData}`)
+            return
+        }
+        return await response.json();
+    }
+
+
+
```

Comment:
This function is, in the meantime, already implemented in `static/helpers.js` Can you remove this here and use the other version ?

Reply:
Done in  0d69ae80